cloudify_cli/cli/helptexts.py

```diff
@@ -222,7 +222,14 @@
                        'If the certificates metadata (I.E: the Manager IP ' \
                        'address) from the snapshot does not match the ' \
                        'Manager metadata, the certificates cannot work on ' \
-                       'this Manager and will not be restored'
+                       'this Manager and will not be restored. In case the ' \
+                       'certificates has been restored, the Manager will be ' \
```

Comment:
~In case the certificates has been restored,~
**In the event that the certificates have been restored,**